### DISCUSSION TITLE
limit exceeded, upgraded to pro, still blocked

### DISCUSSION THREAD

**Discussion by AndreasZeiser** [vercel/vercel#7797] [General]
Hi,

one of our projects exceeded the bandwidth. Yesterday we received a notification that our account has been blocked. One of the proposed solution on Vercels website was to upgrade to the pro plan which I did 2 hrs ago but our account is still blocked and the credit card hasn't been charged yet. 

Why does it so long? How can I speed u the process?

**Comment by swarnava** [comment]
Hi Andreas, can you create a support ticket and let us know the case #?

**Comment by AndreasZeiser** [comment]
Hi Swarnava, I created two tickets 00081143 and 00081144. One of the
support members already replied to me and unblocked my account. Also,
they're investigating why the block wasn't automatically removed after I
upgraded to the pro plan.

Am Fr., 13. Mai 2022 um 22:13 Uhr schrieb Swarnava Sengupta <
***@***.***>:
